src/guide/usage.md

```diff
@@ -23,7 +23,8 @@ The first time you run Infection for your project, it will ask you several quest
     "logs": {
         "text": "infection-log.txt",
         "summary": "summary-log.txt",
-        "debug": "debug-log.txt"
+        "debug": "debug-log.txt",
+        "per-mutator": "per-mutator.md"
```

Comment:
all our configuration keys are in camel case https://infection.github.io/guide/usage.html#Configuration-settings

what do you think about `perMutator` to make it consistent?